.github/workflows/reproduire-close.yml

```diff
@@ -10,6 +10,7 @@ permissions:
 jobs:
   stale:
     runs-on: ubuntu-latest
+    if: ${{github.event_name == 'workflow_dispatch' || github.repository == 'nuxt/nuxt.js'}}
```

Comment:
```suggestion
    if: ${{github.event_name == 'workflow_dispatch' || github.repository == 'nuxt/nuxt'}}
```

Reply:
Ah I see I accidentally used the old repository name...